tests/foreman/ui/test_contentview.py

```diff
@@ -2935,7 +2935,7 @@ def test_positive_rh_mixed_content_end_to_end(session, module_prod, module_org):
 
 @pytest.mark.tier3
 @pytest.mark.skipif((not settings.robottelo.REPOS_HOSTING_URL), reason='Missing repos_hosting_url')
-def test_positive_errata_inc_update_list_package(session, default_sat):
+def test_positive_errata_inc_update_list_package(session):
```

Comment:
This should have `target_sat` in the parameter.

Reply:
turns out, it isn't used at all! so removed